packages/web-components/src/theme/set-theme.bench.ts

```diff
@@ -0,0 +1,47 @@
+// eslint-disable-next-line
+// @ts-nocheck
```

Comment:
what's the reason of using ts  when you're turning it off completely ?

Reply:
It was a copy-paste. Just removed it and everything is still fine :)